src/main/resources/application.yaml

```diff
@@ -39,8 +39,13 @@ auth.idam.client.redirectUri: ${IA_IDAM_REDIRECT_URI:http://localhost}
 
 ### dependency configuration
 auth.provider.service.client.baseUrl: ${S2S_URL:http://127.0.0.1:4502}
-core_case_data.api.url: ${CCD_URL:http://127.0.0.1:4452}
+core_case_data_api_url: ${CCD_URL:http://127.0.0.1:4452}
 document_management.url: ${DOCUMENT_MANAGEMENT_URL:http://127.0.0.1:4506}
 idam.s2s-auth.totp_secret: ${S2S_TOTP_SECRET:AAAAAAAAAAAAAAAC}
 idam.s2s-auth.microservice: ${S2S_MICROSERVICE:ia}
 idam.s2s-auth.url: ${S2S_URL:http://127.0.0.1:4502}
+
+appealReferenceSequenceSeed: 50000
+
+test_system_user: ${TEST_SYSTEM_USERNAME}
```

Comment:
So following on from above, these would be `IA_SYSTEM_*`

Reply:
done